engine/classes/Elgg/UpgradeService.php

```diff
@@ -144,9 +176,6 @@ protected function upgradeCode($version, $quiet = false) {
 	 * @return mixed
 	 */
 	protected static function includeCode($file) {
-		// do not remove - some upgrade scripts depend on this
-		global $CONFIG;
```

Comment:
I asked because I don't know if it's needed or not. :) If you don't know, maybe we should leave it for now.